### PR TITLE
Add unit test for Support Bundle creation

### PR DESCRIPTION
**Pull request by cyrilc-pro**

### CODE REVIEW

**Review by ttetzlaff — APPROVED** [review]

File: commands/support_bundle.go [hunk 1]
```diff
@@ -11,19 +10,23 @@ import (
...
 
 const (
-	serverIDFlag        = "server-id"
-	targetServerIDFlag  = "target-server-id"
-	downloadTimeoutFlag = "download-timeout"
-	retryIntervalFlag   = "retry-interval"
-	promptOptionsFlag   = "prompt-options"
-	cleanupFlag         = "cleanup"
-	targetRepoFlag      = "target-repo"
+	serverIDFlag       = "server-id"
```
Comment (ttetzlaff) [inline commands/support_bundle.go:18]: Are you sure we want to make the constants public?

**Review by ttetzlaff — APPROVED** [review]

File: commands/support_bundle.go (hunk 1 above)
Comment (cyrilc-pro) [reply commands/support_bundle.go:18]: That's the only way I found to test most of the command code in integration tests.

Comment (ttetzlaff) [reply commands/support_bundle.go:18]: hard code the values in the test.

Comment (cyrilc-pro) [reply commands/support_bundle.go:18]: Right, I'll do that

**Pull request merged**